.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -28,10 +28,10 @@ body:
     validations:
       required: true
     attributes:
-      label: What is version of Home Assistant Core has the issue?
+      label: What version of Home Assistant Core has the issue?
       placeholder: core-
       description: >
-        Can be found in the Configuration panel -> Info.
+        Can be found in: Configuration panel -> Info.
```

Comment:
Bonus: This accepts markdown, so we can add a My home assistant link/button here.

https://my.home-assistant.io